Source/Core/CommandLineOptions.cs

```diff
@@ -2041,6 +2049,7 @@ public int GetArgumentSeparatorIndex(string argList, int startIndex)
 
      {:random_seed N}
```

Comment:
This is a bit of an API gap, since this sets the Z3 random seed options as well. If you have a method that happens to verify for a non-default random seed, and you want to temporarily set it in source, you can't do that without also enabling the randomization in Boogie.

Reply:
> If you have a method that happens to verify for a non-default random seed, and you want to temporarily set it in source, you can't do that without also enabling the randomization in Boogie.

You can, you just set this attribute.

The `{:random_seed N}` attribute, whose affect has been modified in this PR, will have the same affect for a single implementation as setting the /randomSeed option has. That's the whole point of it! I've made the documentation more explicit.

Reply:
I meant if you find that a command like this succeeds:

```
dafny /compile:0 MyFile.dfy /proc:"*MyMethod" /proverOpt:O:smt.random_seed=12345
```

Then adding `{:randomSeed 12345}` won't necessarily verify, since it also perturbs the Boogie translation (and sets `sat.random_seed` as well now). Is there an attribute you can use to set arbitrary solver options per-procedure?

Reply:
I see. 

> Is there an attribute you can use to set arbitrary solver options per-procedure?

I don't think so.

Reply:
Okay, I'm happy moving forward, thanks for clarifying.